oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -185,3 +186,33 @@ def change_power_state(uuid):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@computer_system.route(
+    "/redfish/v1/Systems/<uuid>", methods=["DELETE"])
+def remove_computer_system(uuid):
+    """Removes a specific System
+
+        Args:
+            uuid: The System ID.
+    """
+    # Deletes server profile for given UUID
+    response = g.oneview_client.server_profiles.delete(uuid)
+
+    if response is True:
+        return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
I dont sure about whether omition of mimetype here works good for all possible clients.
All our application is using `mimetype="application/json"`, but if no problem omiting this, ok. 
The status is "no content", so maybe works good for all situations. =)